src/pages/billing/Project.js

```diff
@@ -562,7 +562,7 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
             }, ['Remove Billing Account'])
           ])
         }, [
-          h(Link, { 'aria-label': 'Billing account menu' }, [
+          h(Link, { 'aria-label': 'Billing account menu', style: { display: 'flex', alignItems: 'center' } }, [
```

Comment:
I see multiple `div`s with `alignItems: 'center'` above, which makes me wonder if we could move up `alignItems: 'center'` to a parent node so we don't have to individually `style` children, which helps with maintainability.

Reply:
I played around a bit but couldn't accomplish the simplification I'd hoped to do 🤷🏼

Reply:
Yeah I had the same problem when I originally implemented it. Not sure what the issue is :(